src/video/cameradevice.cpp

```diff
@@ -294,7 +294,7 @@ QVector<QPair<QString, QString>> CameraDevice::getRawDeviceListGeneric()
             return devices;
         }
         if (s->iformat->priv_class) {
-            *(const AVClass**)s->priv_data = s->iformat->priv_class;
+            *reinterpret_cast<const AVClass**>(s->priv_data) = s->iformat->priv_class;
```

Comment:
https://www.ffmpeg.org/doxygen/3.3/structAVFormatContext.html#ac4c0777e54085af2f3f1b27130e2b21b says this is `void*`, so this could be `static_cast`. Are you sure all the `reinterpret_cast`s are necessary?